lib/screens/home_statistic/home_statistic.dart

```diff
@@ -128,16 +128,16 @@ class _ContentState extends State<Content> {
       int year = this._date.year;
       int prevDate = year - 100;
       int lastDate = year + 10;
-      DateTime? pickDate = await showMonthPicker(
-        context: context,
-        initialDate: this._date,
-        firstDate: DateTime(prevDate),
-        lastDate: DateTime(lastDate),
-      );
-      if (pickDate != null) {
-        this.setState(() => this._date = pickDate);
-        calculateAndRender(this._date.month.toString(), _ledgerList);
-      }
+      // DateTime? pickDate = await showMonthPicker(
+      //   context: context,
+      //   initialDate: this._date,
+      //   firstDate: DateTime(prevDate),
+      //   lastDate: DateTime(lastDate),
+      // );
+      // if (pickDate != null) {
+      //   this.setState(() => this._date = pickDate);
+      //   calculateAndRender(this._date.month.toString(), _ledgerList);
+      // }
```

Comment:
Could you explain why you comment the code here?

Reply:
can`t build wecount  because using deprecated button in month_picker_dialog

Reply:
Understood! Hope you could follow this up in the future.